Automattic/jetpack
# Transfer Verbum app into into jetpack-mu-wpcom

## Proposed changes:
Moves [Verbum Comments project](https://github.com/automattic/verbum) into Jetpack mu wpcom plugin. It also has some glue code to handle the transition between the deployed and the upcoming versions.

### Other information:


## Does this pull request change what data or activity we track or use?
Not at all. 

## Testing instructions:
* Navigate to `projects/packages/jetpack-mu-wpcom`
* Run `pnpm run build-js` to build Verbum (development) or Run `npm run build-production-js` (production)
* Run `jetpack rsync jetpack mu-wpcom-plugin`  and select your sandbox as the destination (full command example `jetpack rsync mu-wpcom-plugin user@hostname:~/public_html/wp-content/mu-plugins/jetpack-mu-wpcom-plugin/production`
* Make sure you have set `define( 'JETPACK_AUTOLOAD_DEV', true );` in a `mu-plugin/0-sandbox.php`.
* On WPCOM, modify `comment-experience-loader.php` and make `load_verbum_comments` return early.
* The comment experience should continue to function, and it should be Verbum (not Highlander or core's).
* Verify further that the new experience is in charge by checking the path of the `verbum-comment.js` bundle in Network tab.
* Try your best to break the comment experience. Comment as a guest, **comment from Atomic**, and log in using FB.
